android/build.gradle

```diff
@@ -46,8 +46,7 @@ allprojects {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.longtailvideo.jwplayer:jwplayer-core:3.12.1'
-    implementation 'com.longtailvideo.jwplayer:jwplayer-common:3.12.1'
+    implementation 'com.longtailvideo.jwplayer:jwplayer-core:+'
```

Comment:
sme si isti, ze toto chceme spravit? nechceme si momentalnu verziu manazovat sami?

Reply:
aspon hned zistime ak sa to zase rozbije, downgradnut mozme kedykolvek